float32 x
float32 y
float32 z
float32 distance
float32 step
---
casper_auto_msgs/MapObject map_objects